bindings/rascal/models/krr.py

```diff
@@ -74,11 +75,16 @@ def predict(self, managers, compute_gradients=False):
         np.array
             predictions
         """
-        KNM, Y0 = self._preprocess_input(managers, compute_gradients)
+
         if compute_gradients is False:
+            KNM, Y0 = self._preprocess_input(managers, compute_gradients)
             return Y0 + np.dot(KNM, self.weights).reshape((-1))
         else:
-            return np.dot(KNM, self.weights).reshape((-1, 3))
+            rep = self.kernel._representation
+            gradients = compute_sparse_kernel_gradients(rep, self.kernel._kernel, managers.managers, self.X_train._sparse_points, self.weights.reshape((1, -1)))
```

Comment:
Long long line (how did this get past the linter? I had to scroll in the web interface to see the whole thing)

Reply:
there is not python linter at the moment. The black PR is still in limbo.

Reply:
Hmm. Well, if there's no objections, I'll just go ahead and approve that then?